### ISSUE TITLE
Very strange issue, not able to reproduce it in the fiddle but cannot solve it alone.

### ISSUE THREAD

**Issue by gabn88** [Eonasdan/bootstrap-datetimepicker#1428]
See this video:

http://screencast.com/t/jrGpP5z5 

I have entered an initial date in nl format (a.k. 18-01-2016). Then I enable the datetimepicker with options (locale:'nl', format:'L'). The date changes! If I click the date and change it, and click it again, the data constantly changes. Like it thinks it is another format. 

Very strange... I'm also using jquery ui, but not on that element. So that shouldn't be a problem right?

**Comment by patricktoerner** [comment]
I found a fix, not ideal though. Add the option `timeZone: null,`

**Comment by gabn88** [comment]
I just wanted to say THANKS! This does work.

**Comment by patricktoerner** [comment]
Glad it worked! That is a separate issue. I got that when I updated to the newest version. I downgraded to 4.17.37 and it was fine again. I didn't investigate further, but that worked for me!

**Comment by gabn88** [comment]
Okay, I did something else.

I removed the timeZone: null and added: extraFormats: [ 'YYYY-MM-DD' ] so that the complete init code looked like:

$('.datepicker').datetimepicker({
                    locale: current_locale,
                    format: 'L',
                    showTodayButton: true,
		    extraFormats: [ 'YYYY-MM-DD' ]
                });

My backend (a Django installation) sometimes put the <input value=""> in the local format (which corresponds to current_locale), but sometimes as ISO-8601... therefore the extraFormats.. This works super for me!

Thank for helping!
